routes/v1/account.js

```diff
@@ -13,6 +13,31 @@ const validator = require('utils/validator.js');
 
 const COMPLETE = 100;
 
+router.post('/change-name', auth.jwt, safeHandler(async (req, res, next) => {
+    const newName = req.body.newName;
+
+    try {
+        validator.isString(newName);
```

Comment:
Maybe we can add other validators, like no special characters?

Reply:
Oh yeah right didn't looked them alpha+space seems a good idea!

Reply:
Then this should also be added later in the file, otherwise a name that works for signup won't work if changed to it

Reply:
If `validator.isString(req.body.name);` is all we validate at signup then that's all we should do for name changes too.

Unless there's a specific bug where special characters are causing issues, but that's outside the scope of this PR and should be resolved in a separate PR.

Reply:
Okok noted I will update that